.github/workflows/release.yml

```diff
@@ -43,13 +43,42 @@ jobs:
           docker load -i cheqd-did-resolver.tar
           rm cheqd-did-resolver.tar
 
-      - name: Push cheqd-node image
+      - name: Push did-resolver image
         run: |
           docker tag cheqd-did-resolver ghcr.io/${{ github.repository }}:${{ env.RELEASE_VERSION }}
           docker tag cheqd-did-resolver ghcr.io/${{ github.repository }}:latest
+          docker tag registry.digitalocean.com/cheqd/${{ github.repository }}:${{ env.RELEASE_VERSION }}
+          docker tag registry.digitalocean.com/cheqd/${{ github.repository }}:latest
           docker push ghcr.io/${{ github.repository }}:${{ env.RELEASE_VERSION }}
           docker push ghcr.io/${{ github.repository }}:latest
 
+      - name: Install doctl
+        uses: digitalocean/action-doctl@v2
+        with:
+          token: ${{ secrets.DIGITALOCEAN_ACCESS_TOKEN }}
+
+      - name: push image to digitalocean
+        run: |
+          doctl registry login
+          docker push registry.digitalocean.com/cheqd/${{ github.repository }}:${{ env.RELEASE_VERSION }}
+          docker push registry.digitalocean.com/cheqd/${{ github.repository }}:latest
+
+      - name: DigitalOcean App Platform deployment
+        uses: digitalocean/app_action@v1.1.3
+        with:
+          app_name: ${{ github.repository }}
+          token: ${{ secrets.DIGITALOCEAN_ACCESS_TOKEN }}
+          images: '[
+              {
+                "name": "did-resolver",
+                 "image":{
+                  "registry_type": "DOCR",
+                  "repository": "${{ github.repository }}",
+                  "tag": "${{ env.RELEASE_VERSION }}"
+                },
+              }
+            ]'
+
```

Comment:
How does this handle scenarios like stable version is on main, dev/testing versions are on other branches? We should be able to promote when merging to main and all tests pass for going live with the version on main to be running on DO Apps. A bit like the Cloudflare ones that have a staging environment and promote on main.

If we can't make staging/dev environments via DO Apps, that's fine - then I'll just take main.

I believe in those other repos if I recall we use the Github "environments" feature to distinguish.

Reply:
I think we'll have to set up our own DO Apps staging apps for all deployments. Terraform code can remain pretty similar for both environments. 
Moving it back to _draft_ until the functionality is confirmed.